flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorSubmissionTest.java

```diff
@@ -483,12 +419,13 @@ public void testUpdateTaskInputPartitionsFailure() throws Exception {
 			tmGateway.submitTask(tdd, env.getJobMasterId(), timeout).get();
 			taskRunningFuture.get();
 
+			ResourceID producerLocation = new ResourceID("local");
```

Comment:
maybe factor out as class constant

Reply:
I think it would be better to use random `ResourceID.generate()`